localcache.go

```diff
@@ -288,6 +303,32 @@ func (c *Cache) Purge(older time.Duration) error {
 	return nil
 }
 
+func removeEntry(entry string, older time.Duration) error {
```

Comment:
This should be de-duplicated from `Purge()` too.

Reply:
oops, apologies, that was the intention, fixed.